src/control/cmd/dmg/acl_test.go

```diff
@@ -143,6 +143,7 @@ func TestParseACL_EmptyFile(t *testing.T) {
 
 	if result == nil {
 		t.Error("Expected result, got nil")
+		return
```

Comment:
Hmm. Yeah, probably. I was focused on quieting the linter warning. I started to fix this and then noticed that these tests use `t.Errorf()` throughout. Rather than me attacking this piecemeal, maybe you or Dave could revisit these tests to see if they can be refactored a bit?